components/org.wso2.carbon.healthcheck.api.core/src/main/java/org/wso2/carbon/healthcheck/api/core/internal/HealthMonitorServiceComponent.java

```diff
@@ -60,6 +64,36 @@ protected void activate(ComponentContext ctxt) {
         }
     }
 
+    protected void deactivate(ComponentContext context) {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Health Monitor bundle is deactivated");
+        }
+        stopJvmMemoryLogger();
+    }
+
+    private void startJvmMemoryLogger() {
+
+        javaMemoryUsageLogger = new JavaMemoryUsageLogger(HealthCheckConfigParser.getInstance().
+                getMemoryLoggerWait(), HealthCheckConfigParser.getInstance().getMemoryLoggerInterval());
+        if (HealthCheckConfigParser.getInstance().isMemoryLoggerServiceEnabled()) {
+            if (log.isDebugEnabled()) {
+                log.debug("Memory logging service is starting.");
+            }
```

Comment:
We need  below after the successful start, since the debug only shows "starting.' we need to print the result too.
if (log.isDebugEnabled()) {
                log.debug("Memory logging service is started.");
            }

Reply:
Fixed in javaMemoryUsageLogger.start() method

Reply:
if (log.isDebugEnabled()) {
                log.debug("Memory logging service is starting.");
            }
javaMemoryUsageLogger.start();
 if (log.isDebugEnabled()) {
                log.debug("Memory logging service is started.");
            }

Reply:
Fixed